src/System.Windows.Forms/src/System/Windows/Forms/Control.ThreadMethodEntry.cs

```diff
@@ -19,21 +17,21 @@ private class ThreadMethodEntry : IAsyncResult
             internal Control _marshaler;
             internal Delegate _method;
             internal object[] _args;
-            internal object _retVal;
-            internal Exception _exception;
+            internal object? _retVal;
+            internal Exception? _exception;
             internal bool _synchronous;
-            private ManualResetEvent _resetEvent;
+            private ManualResetEvent? _resetEvent;
             private readonly object _invokeSyncObject = new object();
 
             // Store the execution context associated with the caller thread, and
             // information about which thread actually got the stack applied to it.
-            internal ExecutionContext _executionContext;
+            internal ExecutionContext? _executionContext;
 
             // Optionally store the synchronization context associated with the callee thread.
             // This overrides the sync context in the execution context of the caller thread.
-            internal SynchronizationContext _syncContext;
+            internal SynchronizationContext? _syncContext;
 
-            internal ThreadMethodEntry(Control caller, Control marshaler, Delegate method, object[] args, bool synchronous, ExecutionContext executionContext)
+            internal ThreadMethodEntry(Control caller, Control marshaler, Delegate method, object[] args, bool synchronous, ExecutionContext? executionContext)
```

Comment:
```suggestion
            internal ThreadMethodEntry(Control caller, Control marshaler, Delegate? method, object[]? args, bool synchronous, ExecutionContext? executionContext)
```

![image](https://user-images.githubusercontent.com/4403806/102736041-1157f880-4398-11eb-8f41-33fc56bd424f.png)
![image](https://user-images.githubusercontent.com/4403806/102735988-ed94b280-4397-11eb-8a28-8105d6f0de1a.png)